.github/workflows/continuous-integration.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       matrix:
         # This matrix runs tests on iOS sim & Mac, on oldest & newest supported Xcodes
         runner:
-          - macos-13
+          - macos-14
```

Comment:
macos-14 is used for running min Xcode instead of macos-13.  This matches the macOS used in Catapult.